server/events/project_command_runner.go

```diff
@@ -174,6 +174,7 @@ func (p *DefaultProjectCommandRunner) doPlan(ctx models.ProjectCommandContext) (
 	return &models.PlanSuccess{
 		LockURL:         p.LockURLGenerator.GenerateLockURL(lockAttempt.LockKey),
 		TerraformOutput: strings.Join(outputs, "\n"),
+		ChangeSummary:   outputs[len(outputs)-2],
```

Comment:
Question: Is this always the change summary? What if the plan fails, what about other versions of terraform?

Reply:
Thanks for the feedback! These are great questions, and I'm not sure what to do about them. Do you have any suggestions? I suppose I could add more sophisticated parsing of the output that looks for the specific line?

> What if the plan fails?

This is `PlanSuccess`, which I assumed was only for successful plans. Is that assumption incorrect?

> Is this always the change summary? 
> What about other versions of terraform?

Great points. From the integration test results, it looks like the last line is not always the change summary after-all :(
For other versions of terraform, I'm not sure.